test/membrane_framerate_converter_test.exs

```diff
@@ -58,44 +59,40 @@ defmodule Membrane.FramerateConverterTest do
   end
 
   describe "FramerateConverter should" do
-    defp perform_general_test(elements) do
-      pipeline_options = %Pipeline.Options{elements: elements}
-      assert {:ok, pid} = Pipeline.start_link(pipeline_options)
+    defp perform_general_test(structure) do
+      assert pipeline = Pipeline.start_link_supervised!(structure: structure)
 
-      assert_end_of_stream(pid, :sink, :input, 2_000)
-      Pipeline.terminate(pid, blocking?: true)
```

Comment:
don't you want to wait until the pipeline is terminated?

Reply:
It is no longer needed, thanks to `Membrane.Testing.Pipeline.start_link_supervised!/1`